Mage.Tests/src/test/java/org/mage/test/cards/enchantments/SkullclampTest.java

```diff
@@ -25,9 +25,7 @@ public class SkullclampTest extends CardTestPlayerBase {
      * 704.5n.)
      *
      */
-    
-    // This test does not work, but the example works in the game fine.
-    @Ignore
+
```

Comment:
missing `@Test`?

Reply:
Oops, that one didn't actually pass. Thanks for catching. Maybe I should add the test tag but keeps the ignore?

Reply:
Yeah, that sounds right to me.